_posts/2015-05-12-new-lipsum.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog
+category: blog
+published: true
+title: New Lipsum
+---
+
+## A New Post
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent mollis, ante
+eu sollicitudin fringilla, velit mauris faucibus urna, et rutrum quam augue et
+lorem. Cras ut consectetur turpis. Pellentesque eleifend accumsan magna eget
+mattis. Nunc non porttitor risus. Suspendisse et enim felis. Curabitur
```

Comment:
this is a comment on another line in the same paragraph. `Nunc non porttitor risus.` is crappy! remove that.